airbyte-integrations/connectors/source-kafka/src/main/java/io/airbyte/integrations/source/kafka/format/Avro2JsonConvert.java

```diff
@@ -0,0 +1,154 @@
+package io.airbyte.integrations.source.kafka.format;
+
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static java.util.Map.entry;
+
+public class Avro2JsonConvert {
+
+    private ObjectMapper mapper = new ObjectMapper();
+
+    /**
+     * Mapping from avro to Json type
+     * @link https://docs.airbyte.com/understanding-airbyte/json-avro-conversion/#conversion-rules
+     */
+    private static final Map<String, List<String>> AVRO_TO_JSON_DATA_TYPE_MAPPING = Map.ofEntries(
+            entry("null", List.of("null")),
+            entry("boolean", List.of("boolean", "null")),
+            entry("int", List.of("integer", "null")),
+            entry("long", List.of("integer", "null")),
+            entry("float", List.of("number", "null")),
+            entry("double", List.of("number", "null")),
+            entry("bytes", List.of("string", "null")),
+            entry("string", List.of("string", "null")),
+            entry("record", List.of("object", "null")),
+            entry("enum", List.of("string", "null")),
+            entry("array", List.of("array", "null")),
+            entry("map", List.of("object", "null")),
+            entry("fixed", List.of("string", "null"))
+    );
+
+
+    private List<String> avroTypeToJsonType(String avroType) {
+        List<String> jsonTypes = AVRO_TO_JSON_DATA_TYPE_MAPPING.get(avroType);
+        if (jsonTypes == null) {
+            throw new IllegalArgumentException("Unknown Avro type: " + avroType);
+        }
+        return jsonTypes;
+    }
+
+    public JsonNode convertoToAirbyteJson(String avroSchema) throws Exception {
+        Map<String, Object> mapAvroSchema = mapper.readValue(avroSchema, new TypeReference<>() {
```

Comment:
Airbyte provices a helper class that we can use to serialise/deserialise Json - `io.airbyte.commons.json.Jsons.`